src/main/java/com/example/java/maven/cardGamePeterPan/Player.java

```diff
@@ -0,0 +1,86 @@
+package com.example.java.maven.cardGamePeterPan;
+
+import java.util.ArrayList;
+
+public class Player {
+    private final String name;
+    private final ArrayList<Card>hand;
+    private boolean isPlaying;
+
+    public Player(String name) {
+        this.name = name;
+        this.hand = new ArrayList<>();
+        this.isPlaying = true;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<Card> getHand() {
+        return hand;
+    }
+
+    @Override
+    public String toString() {
+        return name + ", with hand " + hand;
+    }
+
+    public void addCardToHand(Card card) {
+        hand.add(card);
+    }
+
+    public void discardCardsFromHand(int firstCardIndex, int secondCardIndex) {
+        hand.remove(secondCardIndex);
+        hand.remove(firstCardIndex);
+    }
+
+    public boolean areCardsMatchingBySuit(int firstCardIndex, int secondCardIndex) {
```

Comment:
is do we have different behavior for these cases: `areCardsMatchingBySuit`, `areCardsMatchingByRank`?
Or can we have only 1 case: `areCardsMatching`? :D